ncm-accounts/src/main/pan/components/accounts/functions.pan

```diff
@@ -42,6 +42,11 @@ function is_user_or_group = {
         );
     };
     pref = "/software/components/accounts";
+    # not all sites use ncm-accounts, e.g. users and groups may be provisioned
+    # via LDAP instead so return now we've confirmed it is a string
+    if (!path_exists(pref)) {
```

Comment:
if the path exists but component is inactive, it should also return true. that way you can include the component, but put it inactive.

Reply:
Strictly true but feels wrong. Is the config in the active component valid? Partial? Typically we disable components during different phases of build so the config is valid but not enforced. That would interact badly with this proposal.

Reply:
ok, after reading the explanation in  #1166 i understand it better now.